src/main/java/edu/ucla/library/prl/harvester/services/HarvestScheduleStoreService.java

```diff
@@ -136,4 +140,22 @@ static HarvestScheduleStoreService createProxy(final Vertx aVertx) {
      */
     @ProxyClose
     Future<Void> close();
+
+    /**
+     * @param aVertx A Vert.x instance
+     * @param aConfig A configuration
+     * @return A connection pool
+     */
+    static Pool getConnectionPool(final Vertx aVertx, JsonObject aConfig) {
```

Comment:
A philosophical question: why put this method here and not in the `impl` class?

Reply:
Since it's a static method and doesn't deal with the implementation details of HarvestScheduleStoreServiceImpl, it didn't seem appropriate to put there. But along those same lines, I can see that it might make more sense to put it in a generic utilities class instead of this interface. However, this service is the only one that directly references a `Pool`, so this felt like a suitable place.